clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/utils/KubeTestUtils.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.it.utils;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import java.util.function.BooleanSupplier;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.ResourceLoader;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.constructor.SafeConstructor;
+
+public abstract class KubeTestUtils {
+
+  private static final int SLEEP_STEP_SECONDS = 5;
+
+  public static TestResourceFile loadYaml(String file) {
+    ResourceLoader resourceLoader = new DefaultResourceLoader();
+    try {
+      InputStream is = resourceLoader.getResource(file).getInputStream();
+      Yaml yaml = new Yaml(new SafeConstructor());
+      Iterable<Object> contentIterable = yaml.loadAll(is);
+      List<Map<String, Object>> content =
+          StreamSupport.stream(contentIterable.spliterator(), false)
+              .filter(Objects::nonNull)
+              .map(KubeTestUtils::coerceManifestToList)
+              .flatMap(Collection::stream)
+              .collect(Collectors.toList());
+      return new TestResourceFile(content);
+    } catch (IOException e) {
+      throw new RuntimeException("Unable to load manifest from file " + file, e);
```

Comment:
Not really important for a test, so feel free to ignore here...but often `UncheckedIOException` is the preferred way to wrap an `IOException` that we're just catching to avoid dealing with checked exceptions.